### ISSUE TITLE
README.md for each formal languages

### ISSUE THREAD

**Issue by TomyYamy** [nickovic/rtamt#24]
Maybe we can put README.md  for each formal languages.
The location is maybe under rtamt/rtamt/grammar/xxx/README.md

**Comment by ANogin** [comment]
Yes, please - would be very helpful. I've been trying to gleam this from papers you cite, and not everything is easy to figure out. E.g. in your example specs in `README.md` you use `out = ...` in your specs, and sometimes you have `spec.declare_var('out', 'float')` and sometimes you do not, and it's all unclear. In general, I have no clue e.g. whether I can introduce my own intermediate variables with `var = ...` + then use `var` later in the spec, but not ever provide the system with the value of `var`.
